Portfolio/index.html

```diff
@@ -33,12 +39,48 @@ <h2>Skills</h2>
         <li>React</li>

```

Comment:
Consider adding a class or id to the 'b' element for easier styling and targeting in CSS.